tools/lint_diff.py

```diff
@@ -52,10 +52,19 @@ def find_branch_point(branch):
         'Failed to find a common ancestor in the last 1000 commits')
 
 
-def find_diff(sha):
+def find_diff(sha, files=None):
     """Find the diff since the given sha."""
+    if files:
+        for file_or_dir in files:
+            assert os.path.exists(file_or_dir), (f"{file_or_dir} doesn't "
+                                                 "exist. Please provide a "
+                                                 "valid path.")
```

Comment:
Not required, but how about:
```suggestion
            msg = f"{file_or_dir} doesn't exist. Please provide a valid path."
            assert os.path.exists(file_or_dir), msg
```